package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "kurakoo",
+  "version": "1.0.0",
+  "description": "![kurakoo-logo](https://user-images.githubusercontent.com/57852378/104982229-3cce1180-5a30-11eb-9d02-3c32878c6a42.png)",
+  "main": "index.js",
+  "scripts": {
+    "start": "concurrently \"cd server && npm run start\" \"cd client && npm start\" ",
+    "dev": "concurrently \"cd server && npm run dev\" \"cd client && npm start\" "
```

Comment:
Please don't use `dev` use again, because if someone wants to run only backend then he can't.

Also, make changes to documentation accordingly!

Reply:
@codewithvk  should I use " kurakoo" instead? 
like "npm run kurakoo"

Reply:
and for the documents  I was thinking of raising another issue cuz in that the backend in point 2 
 it says "the format should be as given in `.env.example` in the server directory itself" and that file does not exist anymore

Both can be dealt with at the same time

Reply:
@Subham142 You can make changes in this PR, I will assign this PR as level 3.

Reply:
@Subham142 for changing CLI, You can change for backend instance of `dev` to `server`